uber/server.py

```diff
@@ -14,10 +14,8 @@ def _add_email():
 class UberShutDown:
     def default(self, *args, **kwargs):
         return render('closed.html')
-    
-    signups = signups.Root()
+
     schedule = schedule.Root()
-    preregistration = preregistration.Root()
```

Comment:
I figured we could leave the schedule online while we're being moved, but we should still turn off everything else.